### PR TITLE
fix: support iOS 16+ for menuItems #3269

### PR DESCRIPTION
**Pull request by hoangduy0610**

### CODE REVIEW

**Comment by hoangduy0610** [comment]
#3269 this issue

**Comment by thaibm** [comment]
Nice change! Really helpful.

**Comment by hoangduy0610** [comment]
@chiaramooney please take a look of this PR

**Comment by chiaramooney** [comment]
I only maintain the Windows code of the module. @Titozzz might be able to help.

**Comment by Titozzz** [comment]
I'm just missing some time but I'll get to it I promise

**Comment by hoangduy0610** [comment]
how is it going, mr @Titozzz ? ^_^ Hope to receive feedback from you soon. My project was blocking by this and I must patch package manually, so I think this PR will be helpful for others. Thank you so much

**Comment by Titozzz** [comment]
Hey! I started cleaning stuff last weekend so hopefully I'll get to it soon! Thanks for being polite and understanding 🤗

Regarding patch-package, yeah that's what we all do, waiting for our PRs to get merged, it's a good practice ✌️. 

(I know it's a little more maintenance so hopefully you ping the maintainer every now and then ❤️)

File: apple/RNCWebViewImpl.m [hunk 1]
```diff
@@ -240,6 +253,37 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecogni
...
+      NSString *menuItemLabel = [RCTConvert NSString:menuItem[@"label"]];
+      NSString *menuItemKey = [RCTConvert NSString:menuItem[@"key"]];
+      UIAction *item = [UIAction actionWithTitle:menuItemLabel image:nil identifier:nil handler:^(UIAction *action) {
+          [self tappedMenuItem:menuItemKey];
+      }];
+      [customMenuActions addObject: item];
+    }
+    UIMenu *customMenu2 = [UIMenu menuWithChildren:customMenuActions];
+
+  // [actions addObject:customMenu];
```
Comment (TheAlmightyBob) [inline apple/RNCWebViewImpl.m:279]: Should this be removed?

Comment (hoangduy0610) [reply apple/RNCWebViewImpl.m:279]: i see, i will update this

Comment (hoangduy0610) [reply apple/RNCWebViewImpl.m]: Updated, please take a look. Thank you so much <3

**Comment by hoangduy0610** [comment]
@TheAlmightyBob I updated the code, please take a look when you available. Thank you so much <3

**Comment by hoangduy0610** [comment]
@TheAlmightyBob @Titozzz Hi there, I'm still waiting for approval.

P/s: Don't be mad at me, I just want to notify you.

**Comment by Titozzz** [comment]
No worries, as long as it's once in a while and friendly it's good to nudge me 🥰

Added to Monday todo

**Comment by Saadnajmi** [comment]
> @TheAlmightyBob @Titozzz Hi there, I'm still waiting for approval.
> 
> 
> 
> P/s: Don't be mad at me, I just want to notify you.

I wonder of this if already fixed by my earlier PR: https://github.com/react-native-webview/react-native-webview/pull/3373

**Comment by hoangduy0610** [comment]
Hi @Titozzz , I confirm that this is already fixed in @Saadnajmi PR. Thank you so much. I will close my PR.

**Pull request closed without merging**